core/block_animations.ts

```diff
@@ -37,6 +34,7 @@ let disconnectGroup: SVGElement|null = null;
 
 /**
  * Play some UI effects (sound, animation) when disposing of a block.
+ *
```

Comment:
This is part of the TsDoc style, not JsDoc. The Google style guide says to use JsDoc but we have departed from the style guide since we are now going to be using TS-specific tools rather than something like tsickle which would require the use of jsdoc comments internally.

The TsDoc style does require this blank line, and eslint will automatically add it if it is missing (if eslint `--fix` option is used). So I think we should go with it in the interest of following the relevant style guide where possible.

Reply:
> This is part of the TsDoc style

Ugh.  But I guess we will have to live with it.